crypto/stateproof/verifier.go

```diff
@@ -26,8 +26,9 @@ import (
 
 // Errors for the StateProof verifier
 var (
-	ErrCoinNotInRange = errors.New("coin is not within slot weight range")
-	ErrNoRevealInPos  = errors.New("no reveal for position")
+	ErrCoinNotInRange    = errors.New("coin is not within slot weight range")
+	ErrNoRevealInPos     = errors.New("no reveal for position")
+	ErrTreeDepthTooLarge = errors.New("state proof tree depth is too large")
```

Comment:
2 nits:
1- can we make the message more accurate? - i.e use the word `participantTree` and `SignatureTree`
2- should we format this error message with the actual values?